source/common/config/filesystem_subscription_impl.h

```diff
@@ -59,6 +59,7 @@ class FilesystemSubscriptionImpl : public Config::Subscription<ResourceType>,
       config_update_available = true;
       callbacks_->onConfigUpdate(typed_resources);
       stats_.update_success_.inc();
+      ENVOY_LOG(debug, "Filesystem config update accepted for {}", path_);
```

Comment:
In the trailers commit you debug printed the entire message. Do you want to do that here? (Don't care just pointing it out).